docs/enhancements/boot-status-reporter.md

```diff
@@ -0,0 +1,129 @@
+---
+title: boot-status-reporter
+authors:
+  - "@nmagnezi"
+creation-date: 2022-09-22
+last-updated: 2022-09-22
+---
+
+# Assisted Boot Reporter
+
+## Summary
+
+Create a new reporting service that runs as early as possible when hosts are booted from disk to collect additional 
+information and send it to `assisted-service`.
+
+## Motivation
```

Comment:
I think our motivation should be 
"No matter what, host should send logs after reboot" from my point status is even less interesting

Reply:
Agree, and we should make sure that these logs don't override/conflict with the current logs gathering we have in place

Reply:
@tsorya, I think this is pretty much what the `Summary` section states. Let me know if you think a change of wording is needed here.

Reply:
@eranco74 By "the current logs gathering" do you mean the logs from the discovery phase? 
Based on [send_logs.go](https://github.com/openshift/assisted-installer-agent/blob/584421642dfb3cbd32ef9d81fcf3be91debe164c/src/logs_sender/send_logs.go#L113), it uses the `V2UploadLogs`. Should we use something else for this?